### PR TITLE
feat(tests): add ability for tests to run in browser on travis

### PR DESCRIPTION
**Pull request by jeffbcross**

### CODE REVIEW

**Comment by jeffbcross** [comment]
I amended my commit to fix an issue with the name of the "postinstall" script that is responsible for installing selenium and chromedriver dependencies. s/post_install/postinstall

**Comment by jeffbcross** [comment]
Looking into why Chrome is crashing on Travis now...

**Comment by joeltine** [comment]
This PR is the first step to resolve #379

**Comment by jeffbcross** [comment]
Looks like Travis is good to go (with some love to fix or ignore the failing tests). Passing the baton to @joeltine from here

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,28 @@
...
   - git clone https://github.com/google/closure-compiler.git
   - cd closure-compiler
   - ant jar
   - cd ../closure-library
+  - npm install
 
 script:
-  - java -jar ../closure-compiler/build/compiler.jar -O ADVANCED --js='**.js' --js='!**_test.js' --js='!**_perf.js' --js_output_file=$(mktemp)
\ No newline at end of file
+  - ./node_modules/.bin/webdriver-manager start & sleep 5; ./node_modules/.bin/protractor protractor.conf.js
```
Comment (joeltine) [inline .travis.yml]: What's this sleep 5 for?

Comment (jeffbcross) [inline .travis.yml]: To give selenium server time to start up before protractor runs.

Comment (joeltine) [inline .travis.yml]: so these commands are synchronous?

File: package.json [hunk 2]
```diff
@@ -2,6 +2,10 @@
   "name": "google-closure-library",
   "description": "Google's common JavaScript library",
   "version": "20150609.0.0",
+  "scripts": {
+    "all_tests": "./node_modules/.bin/protractor protractor.conf.js",
+    "postinstall": "./node_modules/.bin/webdriver-manager update"
```
Comment (joeltine) [inline package.json]: I can delete both of these scripts right?

**Comment by joeltine** [comment]
Canonical is now https://github.com/google/closure-library/pull/528. Thanks Jeff!

**Pull request closed without merging**